src/xtgeo/grid3d/_gridprop_import_xtgcpprop.py

```diff
@@ -14,7 +14,7 @@
 logger = xtg.functionlogger(__name__)
 
 
-def import_xtgcpprop(self, mfile, ijrange=None, zerobased=False):
+def import_xtgcpprop(mfile, ijrange=None, zerobased=False):
```

Comment:
Docstring outdated - `self` can be removed.

Reply:
Fixed